### ISSUE TITLE
Can't find Picasso jar on Gradle Sync.

### ISSUE THREAD

**Issue by andressantiago** [janrain/jump.android#22]
After cloning and attempting to build the `SimpleDemo` in `Samples` Android Studio complains about not being able to find the Picasso Jar from bintray/jcenter.

```
Error:Could not find picasso.jar (com.squareup.picasso:picasso:2.5.2).
Searched in the following locations:
    http://jcenter.bintray.com/com/squareup/picasso/picasso/2.5.2/picasso-2.5.2.jar
```

**Comment by pbjanrain** [comment]
I just tried that link and was able to download the jar.  Perhaps there was some network or server issues.  Can you please try again?

**Comment by andressantiago** [comment]
@pbjanrain Thanks for the speedy response.

You are absolutely right. It is restricted by my network at the office. (Websense 😢 )

The `jcenter()` replacement I suggested in #23 works for different network configurations. (Including websense ridden ones 😄 )

**Comment by pbjanrain** [comment]
OK, I think that makes sense.  I'll pull this change into our next release (unfortunately I just released an update before seeing this).  I'll leave this issue open for now in case anyone else runs into this issue.  Thanks.

**Comment by pbjanrain** [comment]
Resolved in 7.0.2 release.
